lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -748,7 +762,10 @@ outputs =
   where
     _get :: HasType (NonEmpty (AddressAmount t)) s => s -> [AddressAmount t]
     _get = NE.toList . view typed
-    _set :: HasType (NonEmpty (AddressAmount t)) s => (s, [AddressAmount t]) -> s
+    _set
+        :: HasType (NonEmpty (AddressAmount t)) s
+        => (s, [AddressAmount t])
+        -> s
```

Comment:
To be honest, I don't think this makes it any more readable here :/ ... 
I think we should be able to use our judgement to know when to deviate from the guidelines. The line was `81` chars.... :| ...

At the very least, I'd favor consistency between `_get` and `_set` signatures than blindly applying the line-length rule and therefore, also revise the `_get` signature. Now, we have two different styles side-by-side which makes it worse for the reader.

Reply:
@KtorZ wrote:

> I'd favor consistency between _get and _set signatures

I've updated that commit to make `_get` and `_set` consistent with one another. Hopefully the result is a sensible balance. But of course let me know if you'd prefer something else.